pkg/core/perceptor.go

```diff
@@ -167,6 +175,22 @@ func NewPerceptor(config *Config, timings *Timings, scanScheduler *ScanScheduler
 	return perceptor, nil
 }
 
+// getBlackDuckHosts will get the list of Black Duck hosts
+func getBlackDuckHosts(config *Config) (map[string]*Host, error) {
+	password, ok := os.LookupEnv(config.BlackDuck.ConnectionsEnvironmentVariableName)
```

Comment:
Same as the model version of the function.  Just confusing variable?

Reply:
renamed password to connectionstrings